erpnext/stock/doctype/serial_and_batch_bundle/serial_and_batch_bundle.py

```
@@ -312,10 +329,39 @@ def validate_serial_nos_duplicate(self):
+			"Purchase Receipt",
+			"POS Invoice",
+			"Subcontracting Receipt",
+		]:
+			try:
+				voucher_details = frappe.db.get_value(
+					self.voucher_type, self.voucher_no, ["is_return", "return_against"], as_dict=True
+				)
+				if voucher_details and voucher_details.get("is_return"):
+					return True
+			except Exception:
+				pass
```

Ruff (0.14.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)